src/index.ts

```diff
@@ -1796,12 +1800,11 @@ async function displayActivePool(poolParams: PoolParams|PoolParamsP3|PoolParamsN
   let activateButtonContainer = newPool.querySelector("#activate") as HTMLElement
   let activateButton = newPool.querySelector(".activate") as HTMLElement
   let harvestSection = newPool.querySelector(".harvest-section") as HTMLElement
-  
+
   if(wallet != disconnectedWallet) {
     let isAccountRegistered = (await poolParams.stakingContractData.contract.storageBalance()) != null;
 
-    
-    if(!isAccountRegistered) {
+    if(!isAccountRegistered && !newPool.classList.contains("inactive-pool")) {

```

Comment:
This was added into a function called `displayActivePool`. So the added condition should always return false

Reply:
yeah you are right.I have removed it 
also the activate btn being display on ended farms bug is already fixed (for which I added this line of code), I checked the code for that

Reply:
For some reason, it seems that line fixes the issue, since now the activate button is showing. Can you check why an inactive pool is getting into that code? Also we have to find a proper way to fix that code. If you're having troubles with this, let me know

Reply:
Yeah i also just ran the code and checked that, I have fixed the issue but not 100% sure about the problem and fix ,so if you could please test it once and let me know if it is still an issue, it would be very helpful for me 
thanks.